### PR TITLE
initial Husky configuration

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,5 +101,26 @@
...
+          "lower-case",
+          "pascal-case"
+        ]
+      ]
+    }
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run test:eslint",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```
Comment (lalli-flores) [inline package.json]: A bit out of scope, but is there a reason why we're not running tests `pre-push`?

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1 @@
-// Testing bundle works with webex and component libraries
-import webex from 'webex';
-import WebexSDKAdapter from '@webex/sdk-component-adapter';
```
Comment (lalli-flores) [inline src/index.js]: Why are this changes here? I think you need to rebase your branch from master again

File: package.json [hunk 3]
```diff
@@ -40,12 +40,15 @@
...
     "@babel/preset-react": "^7.8.3",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
     "@semantic-release/changelog": "^5.0.0",
     "@semantic-release/git": "^9.0.0",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "clean-webpack-plugin": "^3.0.0",
+    "conventional-changelog-angular": "^5.0.6",
```
Comment (lalli-flores) [inline package.json]: Why are you installing this explicitly? This already gets installed as part of `@commitlint` packages

Comment (akoushke) [reply package.json]: ohhh thats why package.locked didnt change....

Comment (akoushke) [reply package.json]: wait,are you sure? it didnt add it for me. though I didnt install @commitlint. I installed cli and config and then this.

**Comment by taymoork2** [comment]
@lalli-flores is right chore is what you'd use here

File: package.json (hunk 1 above)
Comment (akoushke) [reply package.json]: with github already running the tests for every push, i find it a bit pointless to have this script. with that being said, I don't mind adding a `pre-push` script. but not yet at least. maybe after adding some tests to this repo.

Comment (lalli-flores) [reply package.json]: I don’t think it’s pointless for the `pre-push` commit to run. Having the developer running tests and making sure they work before pushing to Github (specially if a PR is open) would save us from running unnecessary build in our CI. Freeing up resources is important as the project grows. It would be a good standard across all our projects

Comment (akoushke) [reply package.json]: will add it when we come with the testing environment.

File: package-lock.json [hunk 4]
```diff
@@ -1141,33 +1293,53 @@
         "base64-js": "^1.3.0"
       }
     },
+<<<<<<< Updated upstream
```
Comment (lalli-flores) [inline package-lock.json]: There are Git conflict markers on `package-lock.json`
